### PR TITLE
Recherche rapid

### PR DESCRIPTION
**Pull request by AmjadLille**
i did some changes in code:
- rename the class of cards to be searchBarRapid to be more clear.
- rename the folder from cards to search-bar-rapid
- joint two states together to be more clear

merci

### CODE REVIEW

**Comment by Gauthier-Lhomme** [comment]
It's ok for me

**Review by Gauthier-Lhomme — APPROVED** [review]
It's ok for me

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            <a href="https://www.facebook.com/MagicTheGathering.fr/?brand_redir=201120755306">
+              <i className="fab fa-facebook-square" />
+            </a>
+
+            <a href="https://twitter.com/wizards_magic">
+              <i className="fab fa-twitter-square" />
+            </a>
+
+            <a href="https://www.youtube.com/channel/UC8ZGymAvfP97qJabgqUkz4A">
+              <i className="fab fa-youtube" />{' '}
```
Comment (LoicBrassart) [inline src/components/Footer.jsx]: Useless {' '}, to delete

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            </a>
+
+            <a href="https://www.instagram.com/wizards_magic/">
+              <i className="fab fa-instagram" />{' '}
```
Comment (LoicBrassart) [inline src/components/Footer.jsx]: Useless {' '}, to delete

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            </a>
+
+            <a href="https://www.twitch.tv/magic">
+              <i className="fab fa-twitch" />{' '}
```
Comment (LoicBrassart) [inline src/components/Footer.jsx]: Useless {' '}, to delete

File: src/components/Main.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                  src="https://i.ibb.co/MNBtcBk/carte-droite.png"
+                  alt="Carte du Lac de montagne bouillant"
+                  className="Carte"
+                />
+              </li>
+            </ul>
+          </div>
+          <div className="Button">
+            <button className="ButtonGrimoire" type="button">
+              <a href="#9">Ouvrir le grimoire de Zandikar</a>
```
Comment (LoicBrassart) [inline src/components/Main.jsx]: Z*e*ndikar :p

File: src/components/Main.jsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                  alt="Carte de Jace, mage du miroir"
+                  className="Carte"
+                />
+              </li>
+              <li>
+                <img
+                  src="https://i.ibb.co/MNBtcBk/carte-droite.png"
+                  alt="Carte du Lac de montagne bouillant"
+                  className="Carte"
+                />
```
Comment (LoicBrassart) [inline src/components/Main.jsx]: It'd have been easier to host these files locally, wouldn't it ?

File: src/components/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  return (
+    <nav className="navList">
+      <ul>
+        <li>
+          <a href="#a">
+            <img
+              src="https://i.ibb.co/xFMPcrP/magic.jpg"
+              alt="Logo du jeu Magic The Gathering"
+              id="logo"
+            />
```
Comment (LoicBrassart) [inline src/components/Navbar.jsx]: Host your files locally whenever possible

File: src/components/cardlist.jsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
+// eslint-disable jsx-props-no-spreading
+import CardItem from './carditem';
+
+const cardStyle = {
+  display: 'grid',
+  gridTemplateColumns: 'repeat(5, 1fr)',
+  gridGap: '1rem',
+  justifyItems: 'center',
+};
```
Comment (LoicBrassart) [inline src/components/cardlist.jsx]: Be consistent on the styling method on your whole project

File: src/components/search-rapid/search-bar-rapid.jsx [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+          {cards.length > 0 && (
+            <button
+              type="button"
+              className="btn-clear"
+              onClick={this.clearCards}
+            >
+              Clear
+            </button>
+          )}
+          {/* {this.props.cards.length > 0 && <Redirect to="/cartes" />} */}
```
Comment (LoicBrassart) [inline src/components/search-rapid/search-bar-rapid.jsx]: Dead code

File: src/components/cardlist.jsx [hunk 9]
```diff
@@ -0,0 +1,28 @@
+// eslint-disable jsx-props-no-spreading
```
Comment (LoicBrassart) [inline src/components/cardlist.jsx]: Nope, that's not how you work with ESLint :p

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some blocking comments, plus:
- carditem.jsx should be named CardItem.jsx
- Same for cardlist -> CardList
- There are (at least!) three methods of styling on this PR: CSS, SCSS and CSS incrustation

File: src/components/Footer.jsx (hunk 1 above)
Comment (AmjadLille) [reply src/components/Footer.jsx]: done

File: src/components/Footer.jsx (hunk 2 above)
Comment (AmjadLille) [reply src/components/Footer.jsx]: done

File: src/components/Footer.jsx (hunk 3 above)
Comment (AmjadLille) [reply src/components/Footer.jsx]: done

File: src/components/Main.jsx (hunk 4 above)
Comment (AmjadLille) [reply src/components/Main.jsx]: done

File: src/components/Main.jsx (hunk 5 above)
Comment (AmjadLille) [reply src/components/Main.jsx]: done

File: src/components/cardlist.jsx (hunk 7 above)
Comment (AmjadLille) [reply src/components/cardlist.jsx]: done

File: src/components/search-rapid/search-bar-rapid.jsx (hunk 8 above)
Comment (AmjadLille) [reply src/components/search-rapid/search-bar-rapid.jsx]: done

File: src/components/cardlist.jsx (hunk 9 above)
Comment (AmjadLille) [reply src/components/cardlist.jsx]: done

File: src/components/Navbar.jsx (hunk 6 above)
Comment (AmjadLille) [reply src/components/Navbar.jsx]: done

**Review by LoicBrassart — APPROVED** [review]
Much better, No More Evil ;-)

**Pull request merged**